libs/core/src/lib/list/list-navigation-item/list-navigation-item.component.ts

```diff
@@ -113,7 +116,18 @@ export class ListNavigationItemComponent implements AfterContentInit, AfterViewI
     private _dir: 'ltr' | 'rtl' | null = 'ltr';
 
     /** @hidden */
-    constructor(private _elementRef: ElementRef, @Optional() private _rtlService: RtlService) {}
+    private readonly _renderer2 = inject(Renderer2);
+
+    /** @hidden */
+    constructor(private _elementRef: ElementRef, @Optional() private _rtlService: RtlService) {
+        effect(() => {
+            if (this._condensed()) {
+                this._renderer2.addClass(this._elementRef.nativeElement, 'fd-list__navigation-item--condensed');
+            } else {
+                this._renderer2.removeClass(this._elementRef.nativeElement, 'fd-list__navigation-item--condensed');
+            }
```

Comment:
host metadata would be preferable

Reply:
Nope. It will not react on the changes because host stuff is not from this cdr context rather than parent one (which is not vertical nav component but example component)

Reply:
Yes